designer/src/main/java/org/restcomm/connect/rvd/RvdConfiguration.java

```diff
@@ -214,6 +214,7 @@ public static RvdConfig loadRvdXmlConfig(String pathToXml) {
             XmlParser xml = new XmlParser(pathToXml);
             rvdConfig.setAllowedCorsOrigins(xml.getElementList("/rvd/corsWhitelist/origin"));
             rvdConfig.setUssdSupport(xml.getElementContent("/rvd/ussdSupport"));
+            rvdConfig.setExternalServiceTimeout(xml.getElementContent("/rvd/externalServiceTimeout"));
```

Comment:
@croufay , in the event that  #202 is indeed a valid problem (please check my latest comments there) `externalServiceTimeout` is already lazily initialized here:

https://github.com/RestComm/Restcomm-Connect/issues/2410#issuecomment-325275040

Are you sure it's empty ?

Reply:
@otsakir, the value in the rvd.xml is always empty so we default to 5000. We I checked the code I noticed that we are not actually getting the value from rvd.xml so I added a patch for that.

Reply:
Sorry @croufay , i pasted wrong reference. Here is what i meant:
https://github.com/RestComm/visual-designer/blob/edabbdda4c0cbfdc286e6c913c2f737728f3a287/designer/src/main/java/org/restcomm/connect/rvd/RvdConfiguration.java#L292-L292

The getter does the job lazily.